packages/@aws-cdk/region-info/build-tools/generate-static-data.ts

```diff
@@ -0,0 +1,27 @@
+import fs = require('fs-extra');
+import path = require('path');
+import facts = require('./static-data-facts');
+
+async function main(): Promise<void> {
+  const lines = [
+    "import { IRegionInfo } from './iregion-info';",
```

Comment:
`iregion-info` as a file name. no...

Reply:
Suggest better? Can't merge this declaration in "region-info" because otherwise the code generation will depend on the code it generates to compile...